components/storage/blockdevices/COMPONENT_SPIF/README.md

```diff
@@ -0,0 +1,42 @@
+# SPI Flash Driver
```

Comment:
Should live in handbook.

Reply:
Again I agree and will be updated on the next commit tomorrow.